docs/source/articles/integrate-payment-tenders.html.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Integrate Payment Tenders
+excerpt: TODO
+created_at: 2019/10/10
+---
+
+Integrate Payment Tenders
+======================================================================
+
+Workarea provides two default tender types: store credit and credit card, although the credit card implementation requires customization to communicate with a credit card payment service provider in production.
+Workarea provides plugins which fully implement the credit card tender type, as well as plugins that extend the platform with additional tender types (e.g. gift card, PayPal).
+( Browse [Workarea plugins tagged "Payment"](https://plugins.workarea.com/plugins?tags=Payment) to see all such plugins. )
```

Comment:
Can you omit the plugins.workarea.com link? Not all of those are up-to-date and/or open-source. We need to do a bunch of work on that site.